src/main/java/com/easypost/model/Rate.java

```diff
@@ -334,4 +335,23 @@ public String getCarrierAccountId() {
     public void setCarrierAccountId(final String carrierAccountId) {
         this.carrierAccountId = carrierAccountId;
     }
+
+
```

Comment:
Single space instead of double

Reply:
Can a formatter be set up to avoid stuff like this in the future? I'm a fan of prettier and it looks like they have a plugin for Java